src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -255,6 +259,20 @@ const auto params_Int32 = testing::Combine(
         testing::Values(emptyFusingSpec),
         testing::ValuesIn(additionalConfigFP32()));
 
+const auto params_Int32_Prod = testing::Combine(
+        testing::Combine(
+            testing::ValuesIn(axes()),
+            testing::Values(ov::test::utils::OpType::VECTOR),
+            testing::ValuesIn(keepDims()),
+            testing::Values(ov::test::utils::ReductionType::Prod),
+            testing::Values(ElementType::i32),
+            testing::Values(ElementType::undefined),
+            testing::Values(ElementType::undefined),
+            testing::ValuesIn(inputShapes_Int32_Prod_dyn)),
+        testing::Values(emptyCPUSpec),
+        testing::Values(emptyFusingSpec),
+        testing::ValuesIn(additionalConfigFP32()));
```

Comment:
Better rename the test case to `native_int32`, so that we can extend it with other modes in the future, just by adding the to the specific parameters range.

Reply:
Applied. Thanks Maksim!